rapids-cmake/rapids-cython.cmake

```diff
@@ -0,0 +1,33 @@
+#=============================================================================
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#=============================================================================
+include_guard(GLOBAL)
+
+find_package(PythonExtensions REQUIRED)
```

Comment:
Move these find_packages to cython/init.cmake and have `init` be a macro.

The `rapids-<component>.cmake` files should all be side-effect free

Reply:
Actually move everything before the `includes` to the init.cmake macro.